app/Models/User.php

```diff
@@ -48,4 +49,8 @@ public function course()
     {
         return $this->belongsToMany(Course::class, 'CourseUser', 'user_id', 'course_id');
     }
+    public function sendEmailVerificationNotification()
+    {
+        $this->notify(new VerifyEmail); // my notification
```

Comment:
$this->notify(new VerifyEmail);
=>
`$this->notify((new VerifyEmail)->onQueue('default'));`